pkg/intermediate/aggregate.go

```diff
@@ -304,7 +309,13 @@ func (a *AggregationProcess) aggregateRecords(incomingRecord, existingRecord ent
 			}
 			// Update the corresponding source element in antreaStatsElement list.
 			if fillSrcStats {
-				existingIeWithValue, _ := existingRecord.GetInfoElementWithValue(antreaSourceStatsElements[i])
+				name := antreaSourceStatsElements[i]
+				var existingIeWithValue *entities.InfoElementWithValue
+				if isIntraNode {
+					existingIeWithValue, _ = existingRecord.GetInfoElementWithValue(name)
+				} else {
+					existingIeWithValue, _ = existingRecord.GetInfoElementWithValue(getReverseInfoElementName(name))
```

Comment:
Why is this needed? In the case of inter node flows, flows in conntrack module on both nodes follow the same directions.

Reply:
you mean that for flow record from src and flow record from dst, if they represent the same flow, they should have same `packetDeltaCount` and `packetTotalCount`? 
I was under impression that the `packetDeltaCount` in src record should be the same as `reversePacketDeltaCount` in dst record.

Reply:
Yup it is the same packet that is used to create conntrack flow on both nodes, so its direction is considered as original direction and the other direction is reverse direction. You should dump the conntrack flows in the case of inter-node flow scenario and see this.